packages/baseai/src/init/index.ts

```diff
@@ -235,7 +235,7 @@ function displayOutro({ calledAsCommand }: { calledAsCommand: boolean }): void {
 
 async function updateGitignore(): Promise<void> {
 	const gitignorePath = path.join(process.cwd(), '.gitignore');
-	const gitignoreEntry = '# baseai\n**/.baseai/\n';
+	const gitignoreEntry = '# baseai\n**/.baseai/\n\n# Env file\n.env\n';
```

Comment:
I believe it's not ideal to group these in `gitignoreEntry` since we later check for specific `.baseai` entry in `.gitignore` before adding it to the `.gitignore`. 

Typically, if a `.gitignore` exists, it already includes the `.env` file. Adding both `.env` and `.baseai` together might cause the check for `.baseai` to fail because it's unlikely that both entries are next to each other and this can end up adding `.baseai` twice along with `.env` in `.gitignore`.